### PR TITLE
Cp 24 Banner Image Upload

### PR DESCRIPTION
**Pull request by aatbip**
Task/Title
[Banner Image](https://outsidetech.atlassian.net/browse/CP-24?atlOrigin=eyJpIjoiMDIyM2ViZGI1YmU3NDg1YjhjZTE1YjBmMGE0MzNhZDgiLCJwIjoiaiJ9)

#### The main changes are

- [x] added banner image upload feature
- [x] added image cropper
- [x] added drag and drop support

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -28,6 +28,10 @@
         "properties": "always"
       }
     ],
-    "no-unused-vars": "error"
+    "no-unused-vars": "off",
```
[the remark this answers is not in the document]
Comment (aatbip) [reply .eslintrc.json]: I have set this to "warn" in 79f4e60684a557a50506e11e43bccc0c3f006dee commit. However, eslint is throwing warning for variables in interface definitions as well.

File: src/app/page.tsx [hunk 2]
```diff
@@ -1,29 +1,20 @@
...
-    result.client = await copilotAPI.getClient(searchParams.clientId)
-  }
-
-  if (searchParams.companyId && typeof searchParams.companyId === 'string') {
-    result.company = await copilotAPI.getCompany(searchParams.companyId)
-  }
-
-  return result
-}
+import ClientHomeInterface from './components/ClientHomeInterface'
```
[the remark this answers is not in the document]
Comment (aatbip) [reply src/app/page.tsx]: Fixed in 79f4e60684a557a50506e11e43bccc0c3f006dee commit.

File: src/components/ImagePicker/ImagePicker.tsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+        d='M14.2007 8.00075C14.2007 8.33195 13.9319 8.60075 13.6007 8.60075H8.60075V13.6007C8.60075 13.9319 8.33195 14.2007 8.00075 14.2007C7.66956 14.2007 7.40076 13.9319 7.40076 13.6007V8.60075H2.40078C2.06958 8.60075 1.80078 8.33195 1.80078 8.00075C1.80078 7.66956 2.06958 7.40076 2.40078 7.40076H7.40076V2.40078C7.40076 2.06958 7.66956 1.80078 8.00075 1.80078C8.33195 1.80078 8.60075 2.06958 8.60075 2.40078V7.40076H13.6007C13.9319 7.40076 14.2007 7.66956 14.2007 8.00075Z'
+        fill='#0E0E10'
+      />
+    </svg>
+  )
+}
+
+const UploadIcon = () => {
+  return (
+    <svg
```
[the remark this answers is not in the document]
Comment (aatbip) [reply src/components/ImagePicker/ImagePicker.tsx:28]: This is fixed in #1 PR.

File: src/components/select/Select.tsx [hunk 4]
```diff
@@ -0,0 +1,67 @@
+'use client'
+
+import { FC, ReactNode, useState } from 'react'
+import When from '../hoc/When'
```
[the remark this answers is not in the document]
Comment (aatbip) [reply src/components/select/Select.tsx]: This is fixed in #1 PR.

**Review by sazanrjb — APPROVED** [review]

**Pull request merged**